app/views/course/assessment/answer/forum_post_responses/_forum_post_response.json.jbuilder

```diff
@@ -2,7 +2,7 @@
 json.fields do
   json.questionId answer.question_id
   json.id answer.acting_as.id
-  json.answer_text format_ckeditor_rich_text(answer.answer_text)
+  json.answer_text answer.answer_text
```

Comment:
since it's actually not needed in the first place to be there. With it or without it, the answer's text will still be properly formatted on the page.

while it doesn't impact the text formatting, it creates the inconsistencies while doing the autosave (in case of the successful autosaving,
- the answer being sent from the FE to BE will have the non-formatted text such as `\nbsp`, `\n` etc., 
- the answer being saved inside the DB will still have such characters, but 
- the answer being sent back from BE to FE after the saving is done will not have such characters since theirs are replaced with this formatting)

This inconsistencies will lead the answer to be deemed as different than the one being stored in DB and hence when there are such characters inside the text answer (for example, trailing whitespaces etc), the indicator will be showing "Saving" since the system thought the answer is unsaved while it's actually already saved, and thus will render the unexpected behaviour such as repeatedly saving the answer.